app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class DishesController < ApplicationController
+
+  def show
+    @dish = Dish.find(params[:id])
+  end
+
+  def update
```

Comment:
Your intuition is right -- a more RESTful solution here could be to use a joins controller, probably naming it DishIngredientsController and using a `create` action for associating a dish and ingredient together (just moving the content you have here into that `create` action), and then a `destroy` action for when you want to remove that association